### DISCUSSION TITLE
Adaptar para a realidade de Portugal

### DISCUSSION THREAD

**Discussion by VicenteMarianito** [RamonSilva20/mapos#2341] [Ideas]
Tenho estado a experimentar esta projeto. Acho-o muito interessante para o meu negócio local.
No entanto o projeto necessita de ser adaptado para a realidade em Portugal para poder utilizar.
Alguém em Portugal também interessado ? ou conhecem alguém ?
Obrigado

**Comment by apoionacional** [comment]
Boa noite Vicente, qual seria as diferenças significativas de realidade?

**Reply by VicenteMarianito** [reply]
Bom dia, em Portugal temos NIF, Código Postal, distritos em vez de CNPJ, CEP, Estados.... Os números de Telemóvel (Fone)  têm 9 dígitos, As Formas de Pagamento, por exemplo não temos PIX temos MBWay que o pagamento é feito por número de telemóvel.
E temos, claro uma "tradução" de alguns termos, por exemplo Cadatro será Registo entre outros termos.
Será que tem mais utilizadores do programa em Portugal ?

**Reply by willph** [reply]
Sim tem sim.

Tem alguns usuários de Portugal.

Alguns eu indiquei o Julio Lobo, para adaptar o sistema a realidade de Portugal.

Caso queira entra em contato com ele só acessar o grupo do MapOs no WhatsApp.

https://chat.whatsapp.com/D571nPLqNq6JgIsmL6Gucl

**Reply by alessandrodesign** [reply]
Já fiz uns ajustes semelhantes a esse, se precisar posso ajudar. +55 91 989932926
